### DISCUSSION TITLE
When is a new release planned?

### DISCUSSION THREAD

**Discussion by FWest98** [dafny-lang/dafny#4582] [Q&A] [answered]
We are soon planning to start teaching Dafny in a course in university, but for us both https://github.com/dafny-lang/dafny/issues/4328 and https://github.com/dafny-lang/dafny/issues/4407 are severe issues that make it impossible to work with version 4.2.0. So we are wondering if a next release is already planned for the upcoming weeks?
If there are no concrete plans yet, we'll have to include instructions on how to force v4.1.0 in VS Code, but that will without a doubt create a lot of confusion for the students so would not be preferred of course.

**Comment by keyboardDrummer** [comment, marked as answer]
We're working on a 4.3 release. I hope we'll release it this week but I can't make any promises.

**Reply by FWest98** [reply]
Thank you for the quick reply! We're only starting in 2-3 weeks, so let's hope something is out by then
